doc/flame/examples/lib/color_effect.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:doc_flame_examples/ember.dart';
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/experimental.dart';
+import 'package:flame/game.dart';
+import 'package:flutter/animation.dart';
+import 'package:flutter/cupertino.dart';
+
+class ColorEffectExample extends FlameGame with HasTappableComponents {
+  bool reset = false;
+
+  @override
+  Future<void> onLoad() async {
+    final ember = EmberPlayer(
+      position: size / 2,
+      size: size / 4,
+      onTap: (ember) {
+        if (reset = !reset) {
+          ember.add(
+            ColorEffect(
+              const Color(0xFF00FF00),
+              const Offset(0.0, 0.6),
+              EffectController(duration: 1.5),
+            ),
+          );
+        } else {
+          ember.add(
+            ColorEffect(
+              const Color(0xFF1039DB),
+              const Offset(0.0, 0.6),
```

Comment:
I wouldn't do like this. As the color keeps changing alternatively and not coming back (rest). Even the motion doesn't seem smooth. I would suggest a change for these two lines to
```dart
const Color(0xFF00FF00),
 const Offset(0.6, 0.0),
```

This would make it come back to original state and then move forward again.